packages/flutter_tools/test/general.shard/build_system/targets/assets_transformers_test.dart

```diff
@@ -0,0 +1,100 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
i don't like the name of this test library. should this not just be in the other library?

Reply:
I did this because

1. https://github.com/flutter/flutter/blob/935775cb74ea1ed567d0a9c2ad61a023034212c6/packages/flutter_tools/test/general.shard/build_system/targets/assets_test.dart doesn't encapsulate its fakes into a group, so there is a bunch of name pollution/shadowing risk there.
2. https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-more-test-files-avoid-long-test-files

However, your suggestion at https://github.com/flutter/flutter/pull/143815#discussion_r1498441655 may allow us to move this back into assets_test.dart without it being too bad.

Reply:
For your consideration, I've tried this with [attempt to move assets_transformers_test into assets_test.dart](https://github.com/flutter/flutter/pull/143815/commits/bb8a2acb64e0f25c38d79f66086928750f6913e5).

However, `globals.processManager` doesn't seem to refer to the same one provided in the override based on the results of the test. Do I do anything glaringly wrong here?

Reply:
Never mind, I messed up the whitespacing in the test pubspec when I copied the test over. See [I enjoy YAML](https://github.com/flutter/flutter/pull/143815/commits/b872910976ec5a06304778db92260aa6ca01b363)